### ISSUE TITLE
AITPS UII issue 

### ISSUE THREAD

**Issue by barrypannell** [ombegov/ITDB-schema#157]
Name: Barry Pannell
 Organization: DHS/MBC Consulting 
 Urgency: High 
 Description of Issue: The IT Portfolio includes an investment that was submitted in the IT Portfolio BY17 and the initial BY18 IT Portfolio reports.  But was omitted in May 2017 from the final BY18 IT Portfolio.  When we try to include the investment in the bY19 IT Portfolio, as 'new' it is not accepted.  error message says the investment appears to be a continuing investment based on UII.  When I add the previous UII and change the status to 'no change' then it rejects saying it was not active in the last submission.  
Submitted via XML upload page.  
Desired result - either allow the investment to be included as either 'new' with blank previous UII or 'no change' and allow previous UII or define an approach to have this investment included.  .   

Error messages from the 2 alternate submissions: 

◦IT Portfolio with UII: 024-000006340 : 
alertError: CurrentUII 024-000006340 appears to be a continuing investment that was submitted in the prior budget year, though the PriorUII in the current submission is blank

◦IT Portfolio with UII: 024-000006340 : 
alertError: PriorUII must correspond to an active investment from the prior budget year submission

**Comment by GihaVillazon** [comment]
Hi Barry, 

I want to make sure I understand the error here. You are adding a new investment, and using the "Prior UII" option correct, and its presenting an error?

Thank You.

**Comment by ojedanico** [comment]
Hey Barry, 
This investment was eliminated by not including it in the final budget.  The system doesn't allow you to bring back an investment that was in use in the past once it was eliminated. Since the system will not allow you to reuse a UII that's been eliminated, you have to use a new UII. 

If you believe this was investment was eliminated in error let me know and I will bring it up with Kelly.

**Comment by barrypannell** [comment]
Ok.
Was this validation tightened up this year?  That the submission could not bring back the investment UII?  If I do not submit an investment in the fall submission do I have the same problem in the winter.  Or is this specific to the fact it did not go in BY18 final?

**Comment by ojedanico** [comment]
The validation was not changed, it has always been the case that any UII eliminated could not be reused.

If the budget is not submitted in the fall, but existed in the President's budget submission (Jan/Feb), there will not be a problem.  When final budget submissions are processed, the system looks back to the last submitted stage (President's Budget) in the previous budget year.  It does not look at the pre-decisional submissions (Sept submission).  Any changes or omissions done in the pre-decisional budget will have no affect on submissions uploaded for the President's budget.

Again let me know if this was done in error and will discuss with Kelly.

**Comment by barrypannell** [comment]
Ok.  I think we probably had the situation where an investment did not get included in the initial submission but then was added for the final budget submission.  I discussed with DHS lead.  We will assign new UIIs since they are non-major.  We have 2 of these situations.  One is definitely the same investment but DHS said just go with new UII.  (History-In may, there was still indecision about the investment being standalone or wrapped into another investment and it ended up temporarily in our system with no funding so it was not included.)
